src/engraving/dom/pitch.cpp

```diff
@@ -34,16 +30,10 @@ namespace mu::engraving {
 
 int PitchList::pitchOffset(int tick) const
 {
-    if (empty()) {
-        return 0;
-    }
-
-    std::vector<int> ticks = muse::keys(*this);
-    auto it = std::upper_bound(ticks.cbegin(), ticks.cend(), tick);
-    if (it == ticks.cbegin()) {
+    const auto it = upper_bound(tick);
+    if (it == cbegin()) {
         return 0;
     }
-    --it;
```

Comment:
It seems that `--it` was not included in the refactored version. Is it still necessary? Otherwise the `it == cbegin()` check might have become unnecessary too.